pkg/name/repository.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const (
 	defaultNamespace = "library"
-	repositoryChars  = "abcdefghijklmnopqrstuvwxyz0123456789_-./"
+	repositoryChars  = "abcdefghijklmnopqrstuvwxyz0123456789_-./:"
```

Comment:
We want the registry to allow `:` not the repository right?

Reply:
Registry delegates the decision making on validity of the "repository" to Repository